src/Repository/JsonGenericCollectionRepository.php

```diff
@@ -21,10 +21,6 @@ class JsonGenericCollectionRepository implements GenericCollectionRepositoryInte
     public function __construct(string $jsonFile)
     {
         $this->jsonFile = $jsonFile;
-
-        if (!file_exists($this->jsonFile)) {
```

Comment:
Why you don't want to create file at the beginning?

Reply:
because i don't want the files, so i turn it off, but it creates it anyway.

Reply:
So you store generic in your repository and that's why you don't want to this file?